src/util/RedirectUtil.js

```diff
@@ -0,0 +1,61 @@
+/*!
+ * Copyright (c) 2015-2018, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+define(function () {
+  var fn = {};
+
+  /**
+   * Redirects the browser to a new URL by directly setting window.location.
+   */
+  fn.setWindowLocationTo = function (url) {
+    window.location = url;
+  };
+
+  /**
+   * Redirects the browser to a new URL without using window.location.
+   */
+  fn.redirectTo = function (url) {
+    var form = this.buildDynamicForm(url);
+    document.body.appendChild(form);
```

Comment:
We shall try to append form to widget main container rather than body.
I think we have general guide line that do not populate `body` as much as possible.

Reply:
I don't know how to append to the Widget container from inside this method. If you do, feel free. 😄 

As is, the form is totally invisible and is only in the DOM for a few ms before `submi()` is called.

Reply:
basically pass `this.$el` from View/Controller to this redirect function.